lightning/pkg/importer/table_import.go

```diff
@@ -672,6 +674,13 @@ func (tr *TableImporter) preprocessEngine(
 		chunkCp     *checkpoints.ChunkCheckpoint
 	}
 
+	memInfo, err := mem.VirtualMemory()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	// set the total memory limit for lightning to 80% of the total memory
```

Comment:
so how 80% come from? other part of the code takes memory, but i am not sure how many they take